themes/bootstrap5/templates/Recommend/SideFacets.phtml

```diff
@@ -34,6 +34,17 @@
 ?>
 <?php if ($results->getResultTotal() > 0 || $hasVisibleCheckboxes): ?>
   <h2><?=$this->transEsc($this->slot('side-facet-caption')->get('Refine Results')) ?></h2>
+  <?php if ($this->multiFacetsSelection): ?>
+    <div class="multi-filters-selection">
+      <div class="apply-filters-selection">
+        <input id="userSelectionMultiFilters" class="userSelectionMultiFilters" type="checkbox">
```

Comment:
It's possible to configure multiple instances of SideFacets to be displayed one after the other (for example, if you want to display some facet fields, then a different control, then more facet fields -- we do this at Villanova to put a date graph into the sidebar). Maybe it would make sense to introduce a counter into this template (e.g. something like `$this->layout()->sideFacetsInstanceCounter = ($this->layout()->sideFacetsInstanceCounter ?? 0) + 1`) to keep track of how many times it is rendered, and only display this control on the first appearance.

If we used that same counter to also control the display of the `<h2>` it would have the added bonus of allowing me to remove a local customization from this template in our local instance at Villanova.

In any case, I bring this up because if we don't limit the controls to render only on the first instance of the module, we'll end up with duplicate IDs in the HTML.

Reply:
Done